### PR TITLE
Bugfix/discussion form

### PR DESCRIPTION
**Pull request by donatelloraphael**

### CODE REVIEW

File: spa/src/views/user/ResetPassword.vue [hunk 1]
```diff
@@ -14,21 +14,24 @@
...
-      <b-form @submit.prevent="passes(resetPassword)">
-        <div class="field-area">
-          <TextInput
-            v-model="email"
-            :label="$t('profile.email')"
-            name="email"
-            type="email"/>
-        </div>
+      <b-form @submit.prevent="passes(resetPassword)" action="placeholder">
+        <input type="email" id="email" name="email" v-model="email" v-show="false">
```
Comment (literakl) [inline spa/src/views/user/ResetPassword.vue:18]: What is a purpose of this construct?

**Review by literakl — COMMENTED** [review]
I would prefer to have a separate MR for each independent issue

**Comment by literakl** [comment]
Is the other commit related to https://github.com/literakl/mezinamiridici/issues/136? I requested a pasword change here:
image

**Comment by literakl** [comment]
You need email in Reset password.

**Comment by donatelloraphael** [comment]
My bad. I thought the change was in reset password. I will change it. I will create separate pull requests for each issue from now on.

**Pull request closed without merging**